src/screens/Departures/Departures.tsx

```diff
@@ -249,61 +246,44 @@ const DeparturesOverview: React.FC<Props> = ({
           )}
         </AccessibleText>
       }
-      // onEndReached={onScrollViewEndReached}
       alertContext={'travel'}
       setFocusOnLoad={true}
     >
       <ScreenReaderAnnouncement message={loadAnnouncement} />
 
       <View style={styles.container}>
-        {data &&
-          fromLocation &&
-          sortStopPlaces(data, fromLocation?.coordinates).map(
-            (stopPlace: StopPlaceDetails) => (
-              <Sections.Section withPadding key={stopPlace.id}>
-                <Sections.GenericClickableItem
-                  onPress={() => {
-                    navigation.navigate('StopPlaceScreen', {
-                      stopPlaceDetails: stopPlace,
-                    });
-                  }}
-                >
-                  <View style={styles.stopPlaceContainer}>
-                    <View style={styles.stopPlaceInfo}>
-                      <ThemeText type="heading__component">
-                        {stopPlace.name}
-                      </ThemeText>
-                      <ThemeText>
-                        {stopPlace.description || 'Holdeplass'}
-                      </ThemeText>
-                      {stopPlace &&
-                        stopPlace.latitude &&
-                        stopPlace.longitude && (
-                          <ThemeText>
-                            {primitiveLocationDistanceInMetres(
-                              stopPlace.latitude,
-                              stopPlace.longitude,
-                              fromLocation?.coordinates.latitude,
-                              fromLocation?.coordinates.longitude,
-                            ) + ' m'}
-                          </ThemeText>
-                        )}
-                    </View>
-                    {getTransportModeSvg(stopPlace.transportMode) && (
-                      <ThemeIcon
-                        style={styles.stopPlaceIcon}
-                        size="large"
-                        svg={
-                          getTransportModeSvg(stopPlace.transportMode) ||
-                          BusSide
-                        }
-                      ></ThemeIcon>
-                    )}
-                  </View>
-                </Sections.GenericClickableItem>
-              </Sections.Section>
-            ),
-          )}
+        {sortAndFilterStopPlaces(data).map((stopPlace: StopPlacePosition) => (
```

Comment:
Kan være greit å gjøre `sortAndFilterStopPlaces(data)` lengre opp (utenfor JSX), og legge bak en useMemo på `data`. Når den ligger her så gjøres det ved hver re-render, som jeg ikke tror er helt nødvendig. (+ data-manipulasjon bør helst også ligge utenfor JSX for å være mer lesbar)

Reply:
Slik? 1167e7e5902580b9d56d9f2228ff4b974e41ac8f